pallets/contracts/src/benchmarking.rs

```diff
@@ -44,6 +45,64 @@ const ENDOWMENT: Balance = 1_000 * POLY;
 
 const SALT_BYTE: u8 = 0xFF;
 
+pub struct BenchmarkContractPolymeshHooks;
+
+impl<T: Config + TestUtilsFn<<T as SysTrait>::AccountId>> pallet_contracts::PolymeshHooks<T>
+    for BenchmarkContractPolymeshHooks
+where
+    T::AccountId: UncheckedFrom<T::Hash> + AsRef<[u8]>,
```

Comment:
I don't think we need this benchmark only implementation, since the important methods are the same now.

Everything else looks good.

Reply:
The only annoying thing is the `TestUtilsFn` bound.